kitsune/search/v2/documents.py

```diff
@@ -33,6 +34,7 @@ class WikiDocument(SumoDocument):
     slug = SumoLocaleAwareKeywordField(store=True)
     doc_id = SumoLocaleAwareKeywordField(store=True)
     is_archived = SumoLocaleAwareBooleanField()
+    is_redirect = SumoLocaleAwareBooleanField()
```

Comment:
This will add many fields without any apparent benefit. These articles should not have been indexed in the first place. My suggestion is to close this PR and not tackle this during query time. These data won't even give us any useful aggs.